spec/src/main/doc/injectionelresolution.asciidoc

```diff
@@ -346,23 +346,14 @@ The behavior of all methods declared by +java.lang.Object+, except for +toString
 
 === Dependency injection
 
-From time to time the container instantiates beans and other Java EE component classes supporting injection. The resulting instance may or may not be a _contextual instance_ as defined by <<contextual_instance>>.
+From time to time the container instantiates beans and other component classes supporting injection. The resulting instance may or may not be a _contextual instance_ as defined by <<contextual_instance>>.
 
-The container is required to perform dependency injection whenever it creates one of the following contextual objects:
+The container is required to perform dependency injection whenever it creates one of the following objects:
 
-* contextual instances of session beans, and
-* contextual instances of managed beans.
+* contextual instances of managed beans, and
+* non-contextual instances of managed beans.
```

Comment:
Original text was:
>The container is required to perform dependency injection whenever it creates one of the following contextual objects:
>
>* contextual instances of session beans, and
>* contextual instances of managed beans.
>
>
>The container is also required to perform dependency injection whenever it instantiates any of the following non-contextual objects:
>
>* non-contextual instances of session beans (for example, session beans obtained by the application from JNDI or injected using +@EJB+),
>* non-contextual instances of managed beans, and
>* instances of any other Java EE component class supporting injection.

it became this text (didn't invented the *non-contextual instances of managed beans* concept, it's introduced in [InjectionTarget](http://docs.jboss.org/cdi/spec/1.2/cdi-spec.html#bm_obtain_unmanaged_instance) section) and this text in Java EE part:

>When running in Java EE, the container must extend the rules defined in <<injection>> and is also required to perform dependency injection whenever it creates one of the following objects:
>
>* contextual instances of EJB session beans,
>* non-contextual instances of EJB session beans (for example, session beans obtained by the application from JNDI or injected using +@EJB+), and
>* instances of any other Java EE component class supporting injection.

I split the list of non-managed between Core and EE, but perhaps lost something in the operation...